truncate_client/src/regions/active_game.rs

```diff
@@ -15,7 +15,7 @@ use hashbrown::HashMap;
 
 use crate::{
     lil_bits::{BattleUI, BoardUI, HandUI, TimerUI},
-    utils::{mapper::MappedBoard, Theme},
+    utils::{mapper::MappedBoard, text::TextHelper, Diaphanize, Theme},
```

Comment:
I remember talking about this word, but I don't remember if we looked it up. Either way, the meaning is fun https://en.wiktionary.org/wiki/diaphanization

Reply:
That word is our word ;) https://github.com/TruncateGame/Truncate/blob/1e35c500465ceed980a6dd3c9abfe12fa0cf6915/truncate_client/src/utils/theming.rs#L86-L96